docs/CONTRIBUTING.md

```diff
@@ -32,3 +32,75 @@ Coming soon!
 After values are pushed the cluster's drone will do a deployment run. Helmfile diff is used targeting `$CLOUD` & `$CLUSTER` before the final deployment of the artifacts.
 
 If you have any meaningful additions to this, please let us know!
+
+## Schema validation
```

Comment:
I would like this CONTRIBUTING file will stay without detailed instructions, so you will have to add own doc file(s) for this, with a reference from the README. Please keep file names lowercase and with dashes (CONTRIBUTING and LICENSE etc are special files).

Reply:
That sounds good, but do you propose to do the same then for GO_TEMPLATING.md which has a reference from CONTRIBUTING.md? Or are these a special exemption I'm not aware of?

Reply:
Maybe we could also use the Github Wiki feature?